nucleus/common/common-util/src/main/java/com/sun/common/util/logging/LoggingConfigImpl.java

```diff
@@ -394,20 +394,25 @@ public synchronized Map<String, String> deleteLoggingProperties(final Collection
     public synchronized Map<String, String> deleteLoggingProperties(final Collection<String> properties,
         final String target) throws IOException {
         LOG.log(Level.INFO, "deleteLoggingProperties(properties={0}, target={1})", new Object[] {properties, target});
-        loadLoggingProperties();
+        loadLoggingProperties(target);
         remove(properties);
-        rewritePropertiesFileAndNotifyMonitoring();
-        return setMissingDefaults(getLoggingProperties());
+        rewritePropertiesFileAndNotifyMonitoring(target);
+        return setMissingDefaults(getLoggingProperties(target));
     }
 
     private void remove(final Collection<String> keys) {
         final Consumer<String> toRealKeyAndDelete = k -> props.remove(LoggingXMLNames.xmltoPropsMap.getOrDefault(k, k));
         keys.forEach(toRealKeyAndDelete);
     }
 
-    private void rewritePropertiesFileAndNotifyMonitoring() throws IOException {
+    private void rewritePropertiesFileAndNotifyMonitoring(final String targetConfigName) throws IOException {
         LOG.finest("rewritePropertiesFileAndNotifyMonitoring");
-        File file = getLoggingPropertiesFile();
+        File file = null;
```

Comment:
You can make it final and remove the redundant null assignment as you always set it in `if` or in `else` block.

Reply:
I have modified the code.